subsys/bluetooth/mesh/net.h

```diff
@@ -362,3 +362,15 @@ struct friend_cred *friend_cred_create(struct bt_mesh_subnet *sub, u16_t addr,
 				       u16_t lpn_counter, u16_t frnd_counter);
 void friend_cred_clear(struct friend_cred *cred);
 int friend_cred_del(u16_t net_idx, u16_t addr);
+
+static inline void send_cb_finalize(const struct bt_mesh_send_cb *cb,
+				    void *cb_data)
+{
```

Comment:
This needs an `if (cb)` check

Reply:
All current callers use `if (cb) send_cb_finalize()`. I can move it into the function if you think that's cleaner - I don't have a strong opinion on it.

Reply:
Not anymore? Or perhaps it's still too early in the morning for me

Reply:
No, early morning for me 😄 One place had it, another didn't. I'll just move it to the function.

Reply:
@trond-snekvik should be fixed now